CONTRIBUTING.md

```diff
@@ -59,6 +59,8 @@ $ npm run db:studio
 
 > **Note** `npm run build` needs to be executed the first time running the project. As it generates a `build/server.js` script that `npm run dev` depends on. Subsequent times, only `npm run dev` is needed to run the app in development mode.
 
+> **Note** If you are running test scripts for the first time, change `DATABASE_URL` in `.env` to `DATABASE_URL="mysql://starchart:starchart_password@127.0.0.1:3306/starchart"` and run `npm run setup` to setup the test database. You can change this back afterward.
```

Comment:
This doesn't need to be done in this PR, but would it be worth to add a new `setup:test` script to do this instead?

Reply:
Perhaps. If someone can write a command to parse the file to change the variable, run `npm run setup`, then change it back.

Reply:
Ah I think we can simply copy the `setup` command to `setup:test`, and override the value for `DATABASE_URL`

Reply:
I think I might just create a PR for this today. Seems simple enough (unless I am missing something)